nil/services/rpc/jsonrpc/txpool_api.go

```diff
@@ -0,0 +1,36 @@
+package jsonrpc
+
+import (
+	"context"
+
+	"github.com/NilFoundation/nil/nil/common/logging"
+	"github.com/NilFoundation/nil/nil/internal/types"
+	"github.com/NilFoundation/nil/nil/services/rpc/rawapi"
+)
+
+type TxPoolAPI interface {
+	GetTxpoolStatus(ctx context.Context, shardId types.ShardId) (uint64, error)
+	GetTxpoolContent(ctx context.Context, shardId types.ShardId) ([]*types.TxnWithHash, error)
```

Comment:
Let's implement something similar to eth: https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-txpool

```
txpool.Content() (map[string]map[string]map[string]*RPCTransaction)
```

Right now we have only "pending" field. But it can be changed in future.